Fixes Email Clients not generating logins/AIC eye toggle

TerryTehCoder opened the pull request:
## About the Pull Request

Fixes the email clients not properly generating logins/usernames which prevented accounts from being created properly at the beginning of the round.

Adds in an AIC Eye Toggle that lets you disable the A.I face that indicates your center of view. It's still there, it just makes the Icon disappear. It can be toggled back on by pressing the command again.

Disables the NTOS computer configuration tool from the subsystems tab. 
It was added when DTraitor fixed everything up and added the base nano_modules to the list.

<!-- Describe the Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Bugfixes are lovely!

The AIC Eye always bothered me and I think having the option to toggle it without explicitly turning it off is great for anything that's not mechanically impacting the round.

## Changelog

:cl:
fix: Fixed email clients not properly generating login/username at round start.
fix: Removes Computer Configuration Tool from subsystems tab.
Add: AIC Eye Toggle
/:cl:

<!-- Please add a short description of why you think these changes would benefit the game.

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

<!--
Examples were changelog entries are optional/not typically required but encouraged:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

As a courtesy, for ported PRs, please include if you have the blessing of the author. While not required, we encourage basic decency in porting others' work. It is also sufficient to note that the author has not expressed displeasure at the idea of their work getting ported.
Please refrain from porting works of authors that have expressed displeasure in having their work ported, thank you.
-->

TerryTehCoder committed:
Fixes Email Clients not generating logins/AIC eyepuppet toggle

MACIEKBAKI commented:
you might want to include a proper changelog

MACIEKBAKI commented on code/modules/mob/observer/freelook/ai/eye.dm:
```diff
@@ -106,3 +106,16 @@
 	eyeobj.acceleration = !eyeobj.acceleration
 	to_chat(usr, "Camera acceleration has been toggled [eyeobj.acceleration ? "on" : "off"].")
+
+/mob/living/silicon/ai/proc/eyepuppettoggle() //Messing with the SEE_INVISIBLE_TAG renders the lighting differently so I'd rather just swap out the icon for nothing and still render lights correctly.
```
`/mob/living/silicon/ai/proc/ai_eye_puppet_toggle()` or `/mob/living/silicon/ai/proc/eye_puppet_toggle()` would be preferable as it keeps the naming scheme

MACIEKBAKI commented on code/modules/mob/living/silicon/ai/ai.dm:
```diff
@@ -32,7 +32,8 @@ var/list/ai_verbs_default = list(
 	/mob/living/silicon/ai/proc/ai_shutdown,
-	/mob/living/silicon/ai/proc/ai_reset_radio_keys
+	/mob/living/silicon/ai/proc/ai_reset_radio_keys,
+	/mob/living/silicon/ai/proc/eyepuppettoggle
```
of course youd change here as necessary

MACIEKBAKI commented on code/modules/tgui/modules/ntos-only/configurator.dm:
```diff
@@ -2,6 +2,7 @@
 	name = "NTOS Computer Configuration Tool"
 	ntos = TRUE
 	tgui_id = "Configuration"
+	available_to_ai = FALSE
```
stealth nerf???

TerryTehCoder replied:
This is mentioned in the PR body and the changelog that got deleted.

MACIEKBAKI replied:
alright but why remove it? wouldn't it make sense the AI can do computer config?

TerryTehCoder replied:
This is a hardware configuration in the AIC subsystems tab. Which would be fine if the AIC actually had any hardware to configure in that sense.  It doesn't. 

I don't believe this program is accessible in the general programs list either?

MACIEKBAKI replied:
ah, that'd make sense, yeah; confused it for general network config

TerryTehCoder commented:
> you might want to include a proper changelog
> 
> 

My changelog was deleted in a prior upload. Thank you for pointing it out.

Updated to include changelog.

TerryTehCoder committed:
Fixes the naming scheme of the Eyepuppet Proc

/mob/living/silicon/ai/proc/eyepuppettoggle to /mob/living/silicon/ai/proc/eye_puppet_toggle
